cmd/gocritic/main.go

```diff
@@ -61,6 +68,10 @@ func main() {
 	cmd.main()
 }
 
+func printVersion() {
+	fmt.Printf("go-critic version: %s\n", version)
```

Comment:
Could you use `log.Printf` here?
For consistency with other sub commands.

It will output `version: ` prefix, so you can just do `log.Println(version)`, for example.